doc/source/examples/incompressible-flow/3d-taylor-green-vortex/3d-taylor-green-vortex.rst

```diff
@@ -0,0 +1,374 @@
+====================
+Taylor-Green Vortex
+====================
+
+This example showcases another canonicall fluid mechanics problem, the Taylor-Green vortex.  This examples features both the traditional matrix-based solver within Lethe ( ``lethe-fluid``) and the matrix-free solver  (``lethe-fluid-matrix-free``) which is more computationnaly efficient, especially for high-order elements (Q2 and above). Post-processing capabilities for enstrophy and kinetic energy are also demonstrated.
+
+
+---------
+Features
+---------
+
+- Solvers: ``lethe-fluid`` (with Q2-Q2) or  ``lethe-fluid-matrix-free`` (with Q2-Q2 or Q3-Q3)
+- Transient problem using ``bdf3`` time integrator
+- Displays the calculation of enstrophy and total kinetic energy
+
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/incompressible-flow/3d-taylor-green-vortex``).
+
+- Parameter file: ``tgv-matrix-based.prm`` and ``tgv-matrix-free.prm``
+- Postprocessing Python scripts: ``plot_dissipation_rate.py`` and ``calculate_dissipation_rate.py``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+The Taylor–Green vortex is an unsteady flow of a decaying vortex, which has an exact closed form solution of the incompressible Navier–Stokes equations in Cartesian coordinates. It is named after the British physicist and mathematician Geoffrey Ingram Taylor and his collaborator A. E. Green [1]. In the present case, we simulate one Taylor-Green vortex at a Reynolds number of 1600 in a domain :math:`\Omega = [-\pi,\pi]\times[-\pi,\pi]\times[-\pi,\pi]` using periodic boundary conditions.
+
+The three velocity components :math:`[u_x,u_y,u_z]^T` and the pressure :math:`p` are specified at time :math:`t=0` by:
+
+.. math::
+
+  u_{x} &= sin(x)*cos(y)*cos(z) \\
+  u_{y} &= -cos(x)*sin(y)*cos(z)\\
+  u_{z} &= 0 \\
+  p &=  \frac{1}{16}*(cos(2x)+cos(2y))*(cos(2z)+2)
+
+In this case, the vortex, which is initially 2D, will decay by generating smaller 3D turbulent structures (vortex tubes, rings and sheets). This decay can be monitored through the total kinetic energy of the system. Since the simulation domain is periodic, it can be demontrated that the time derative of the total kinetic energy :math:`E_k` is directly related to the enstrophy :math:`\mathcal{E}` such that:
+
+
+with
+
+.. math::
+
+  \frac{\mathrm{d}E_K}{\mathrm{d}t} &=  \mathcal{E} \\
```

Comment:
Isn't there a minus missing here?
```suggestion
  \frac{\mathrm{d}E_K}{\mathrm{d}t} &=  \mathcal{E} \\
```

Reply:
Yeah either there or in the definition of the enstrophy, but most likely there.

Reply:
```suggestion
  \frac{\mathrm{d}E_K}{\mathrm{d}t} &=  -\mathcal{E} \\
```